deploy/chart/templates/0000_50_olm_03-services.yaml

```diff
@@ -1,39 +1,43 @@
-{{ if .Values.monitoring.enabled }}
+{{- if or .Values.monitoring.enabled .Values.serviceCa.enabled }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: olm-operator-metrics
+  name: {{ if .Values.serviceCa.enabled }}{{ .Values.serviceCa.olmOperator.serviceName }}{{ else }}olm-operator-metrics{{ end }}
```

Comment:
What if `.Values.serviceCa.olmOperator.serviceName` is blank? Or is that something you'll ensure downstream?
Perhaps the serviceName itself should be in the values.yaml file with a default of `olm-operator-metrics`?

Reply:
👍🏽 Done, PTAL